packages/pyright-internal/src/analyzer/pyTypedUtils.ts

```diff
@@ -21,13 +21,17 @@ export function getPyTypedInfo(fileSystem: FileSystem, dirPath: Uri): PyTypedInf
         return undefined;
     }
 
-    let isPartiallyTyped = false;
     const pyTypedPath = dirPath.pytypedUri;
-
     if (!fileSystem.existsSync(pyTypedPath) || !isFile(fileSystem, pyTypedPath)) {
         return undefined;
     }
 
+    return getPyTypedInfoUnsafe(fileSystem, pyTypedPath);
+}
+
+export function getPyTypedInfoUnsafe(fileSystem: FileSystem, pyTypedPath: Uri) {
```

Comment:
Why is this function named "unsafe"? That's normally a big red flag for callers that the function should be used only under certain circumstances. There should at least be a comment here explaining the name and the circumstances under which this function is safe to call.

Reply:
@erictraut if you prefer other name I can use that other name. any suggestion? I can make it like `getPyTypedInfoWithoutCheckingPyTypePath` or something like that.

Reply:
Choosing a name like this is OK if there's a comment that explains the thinking behind the name. Of course, it's even better if the function's name is sufficiently clear that a comment is no longer needed.

Reply:
@erictraut can you check again? I added comments and changed name to `getPyTypedInfoForPyTypedFile`